tests/test_helpers.py

```diff
@@ -90,7 +90,7 @@ def test_olostep_api():
     website = "https://en.wikipedia.org/wiki/Great_Chicago_Fire"
 
     # Act
-    response = search_with_olostep(website)
+    response = read_webpage_with_olostep(website)
```

Comment:
Let's also add a test for `read_webpage`?

Reply:
Good point! Added test for `read_webpage`